.travis.yml

```diff
@@ -1,15 +1,18 @@
 language: python
 python:
-- 3.5
 - 3.8
 env:
-- TOXENV=django22-drf37
-- TOXENV=django22-drf38
-- TOXENV=django22-drf39
-- TOXENV=django22-drflatest
-- TOXENV=quality
-- TOXENV=docs
-- TOXENV=pii_check
+  - TOXENV=django22-celery44-drf37
+  - TOXENV=django22-celery50-drf37
+  - TOXENV=django22-celery44-drf38
+  - TOXENV=django22-celery50-drf38
+  - TOXENV=django22-celery44-drf39
+  - TOXENV=django22-celery50-drf39
+  - TOXENV=django22-celery44-drflatest
+  - TOXENV=django22-celery50-drflatest
+  - TOXENV=quality
+  - TOXENV=docs
+  - TOXENV=pii_check
```

Comment:
Why the indentation change?  Not a big deal, but makes this inconsistent with the rest of the file.